mpmp/src/view/Frame.java

```diff
@@ -1,133 +1,242 @@
+            //XXX Handle exception
+        }
+
+        gameboard.addGVTTreeRendererListener(new GVTTreeRendererAdapter() {
+            public void gvtRenderingPrepare(GVTTreeRendererEvent e) {
+                setTitle("MPMP - Loading...");
+            }
+
+            public void gvtRenderingCompleted(GVTTreeRendererEvent e) {
+                setTitle("MPMP");
+                SVGDocument svgd = gameboard.getSVGDocument();
+                final Element element = svgd.getElementById("");
+                System.out.println(gameboard.getX() + "  " + gameboard.getSize() + "  " + gameboard.getRenderRect() + "  " + gameboard.getSVGDocument().getRootElement());
+                Graphics g = gameboard.getGraphics();
+                g.drawOval(0, 0, 50, 50);
+
+                gameboard.invalidate();
+            }
+        });
+
+        //Set all layouts
+        setLayout(new BorderLayout());
+        left = new JPanel();
+        chat = new JPanel();
+        bottom = new JPanel();
+        bottomMenu = new JPanel();
+        
+        left.setLayout(new BoxLayout(left, BoxLayout.Y_AXIS));
+        chat.setLayout(new BoxLayout(chat, BoxLayout.Y_AXIS));
+        bottom.setLayout(new BoxLayout(bottom, BoxLayout.X_AXIS));
+        bottomMenu.setLayout(new BoxLayout(left, BoxLayout.X_AXIS));
+
+        //Set testing Buttons
+        JButton player1 = new JButton("Player 1");
+        JButton player2 = new JButton("Player 2");
+        JButton player3=  new JButton("Player 3");
+        JButton player4 = new JButton("Player 4");
+        JButton player5 = new JButton("Player 5");
+        JButton player6 = new JButton("Player 6");
+        JButton trade = new JButton("Tauschen");
+        JButton buyHouse = new JButton("Haus kaufen");
+        JButton buyPlot = new JButton("Strasse kaufen");
+        JButton surrender = new JButton("Aufgeben");
+        JButton btn12 = new JButton("Current Player");
+        JButton btn13 = new JButton("Me is no chat box");
+
+        left.add(player1);
+        left.add(player2);
+        left.add(player3);
+        left.add(player4);
+        left.add(player5);
+        left.add(player6);
+        bottomMenu.add(trade);
+        bottomMenu.add(buyHouse);
+        bottomMenu.add(buyPlot);
+        bottomMenu.add(surrender);
+        bottom.add(bottomMenu);
+        bottom.add(btn12);
+        bottom.add(btn13);
+        
+        /**
+         * Setting everything for the chat prefSize = Width of the chat-frame
+         */
+
+        int prefSize = 200;
+
+        playerList = new JTextPane();
+        playerList.setEditable(false);
+        chat.add(playerList);
+
+        chatField = new JTextField();
+        chatField.setToolTipText("Enter chat here...");
+        chat.add(chatField);
+        chatField.setColumns(10);
+
+        chatBox = new JTextPane();
+        chatBox.setEditable(false);
+        chat.add(chatBox);
+
+        bEndTurn = new JButton("Runde beenden");
```

Comment:
This is my old naming scheme. Either make all buttons conform by adding a `b` in front or by appending `Btn` (or `But`).